aws/templates/createApplicationTemplate.ftl

```diff
@@ -323,209 +323,12 @@
-                                                                                        "TargetGroupArn": { "Ref": "${targetGroupKey}" }
-                                                                                    }
-                                                                                ],
-                                                                                "Conditions": [
-                                                                                    {
-                                                                                        "Field": "path-pattern",
-                                                                                        "Values": [ "${lb.Path}" ]
-                                                                                    }
-                                                                                ],
-                                                                                "ListenerArn" : "${getKey("listenerX" + lbTier.Id + "X" +  lbComponent.Id + "X" + ports[lbPort].Port?c)}"
-                                                                            }
-                                                                        }
-                                                                    [/#if]
-                                                                [/#if]
-                                                            [/#if]
-                                                        [/#if]
-                                                    [/#list]
-                                                [/#if]
-                                            [/#if]
-                                        [/#list]
-                                        [#assign count += 1]
-                                    [/#if]
-                                [/#list]
-                            [/#if]
-                            [#if ecs.Tasks??]
-                                [#list ecs.Tasks?values as task]
-                                    [#if task?is_hash && (task.Slices!component.Slices)?seq_contains(slice)]
-                                        [#if count > 0],[/#if]
-                                        [@createTask tier=tier component=component task=task /]
-                                        [#assign containerListMode = "supplemental"]
-                                        [#list task.Containers?values as container]
-                                            [#if container?is_hash]
-                                                [#include containerList]
-                                            [/#if]
-                                        [/#list]
-                                        [#assign count += 1]
-                                    [/#if]
-                                [/#list]
-                            [/#if]
+
```

Comment:
'ecs' part has been hardcoded in the file names, should we add an additional outer iteration?